.eslintrc

```diff
@@ -102,6 +102,7 @@
     "react/jsx-boolean-value": 0,
     "react/jsx-closing-bracket-location": 1,
     "react/jsx-curly-spacing": 1,
+    "react/jsx-indent": [1, 2],
```

Comment:
This change seems to be causing a conflict with the existing indentation rule that is prevent ESLint from autofixing files.